src/microbe_stage/MicrobeHUD.cs

```diff
@@ -779,19 +783,27 @@ private void UpdateATP(float delta)
             return;
 
         var atpAmount = 0.0f;
-        var capacity = 4.0f;
 
+        // Update to the player's current ATP, unless the player is already disposed
```

Comment:
I'd replace "disposed" with doesn't exist currently, as this comment may be misleading if when the player is disposed is changed.